packages/twenty-server/src/modules/opportunity/standard-objects/opportunity.workspace-entity.ts

```diff
@@ -63,13 +64,14 @@ export class OpportunityWorkspaceEntity extends BaseWorkspaceEntity {
   closeDate: Date | null;
 
   @WorkspaceField({
-    standardId: OPPORTUNITY_STANDARD_FIELD_IDS.probability,
+    standardId: OPPORTUNITY_STANDARD_FIELD_IDS.probability_deprecated,
     type: FieldMetadataType.TEXT,
     label: 'Probability',
     description: 'Opportunity probability',
     icon: 'IconProgressCheck',
     defaultValue: "'0'",
   })
+  @WorkspaceIsDeprecated()
   probability: string;
```

Comment:
should this also be renamed to probabilityDeprecated? (if we want to introduce a new probability field soon?)

Reply:
Im not sure - I don't think we have planned to introduce a new probability field for now, so the vision is to consider this field custom, not to discourage its usage for now. its what I had shared [in the doc](https://docs.google.com/document/d/1BJrg-f5jbCB0LhrvpmrvLW83nAAXFcEMGZcRSRf93Ak/edit) (doc is still to be completed though and migrated to github). but open to rediscuss this

Reply:
Ok!